modules/core/include/opencv2/core/mat.inl.hpp

```diff
@@ -584,6 +584,19 @@ Mat::Mat(const std::initializer_list<_Tp> list)
         return;
     Mat((int)list.size(), 1, traits::Type<_Tp>::value, (uchar*)list.begin()).copyTo(*this);
 }
+
+template<typename _Tp, typename> inline
```

Comment:
> template<typename _Tp, **typename**>

Why we need second "typename" here?

Reply:
Actually, we don't need this second "typename". I fix two files to eraze them.